workshop-content/03-project-work/03.7-slack-notification-lambda-setup/03.7-slack-notification-lambda-setup.md

```diff
@@ -8,13 +8,18 @@ Let's setup the function in serverless. Under functions define a new slack funct
     environment: 
       SLACK_WEBHOOK_URL: ${ssm:/error-slack-webhook-url~true}
     events: 
-      - sns: 
-          arn: 
-            Ref: <SNSName>
-          topicName: ${self:service}-${self:provider.stage}-<SNSName>
+      - sqs:
+          arn:
+            Fn::GetAtt: [<QueueName>, Arn]
+          batchSize: 1
+      - sqs:
+          arn:
+            Fn::GetAtt: [<DeadLetterQueueName>, Arn]
+          batchSize: 1
+          enabled: false
 ```
 
-![image](https://user-images.githubusercontent.com/17308998/151514508-1b57bcd4-a54c-4621-a784-1a17f4d90fdc.png)
+![image](diagram.png)
```

Comment:
Note: would be really cool to see the DLQ working here. What we can do is: 
- Force an error in the slack sending lambda. Deploy changes
- Try generating a dog name that would simulate an error. You should see messages in DLQ
- Fix the slack sending lambda and deploy changes
- in the AWS console in the lambda resource enable the DLQ queue and watch the messages get processed